### PR TITLE
README.md: describe relation with meta-bistro

### PR DESCRIPTION
**Pull request by tobsan**
Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,9 +18,14 @@ This layer depends on:
 * poky (http://git.yoctoproject.org/cgit/cgit.cgi/poky)
 * meta-openembedded (http://cgit.openembedded.org/meta-openembedded/)
 * meta-virtualization (http://git.yoctoproject.org/cgit/cgit.cgi/meta-virtualization),
-if the "process-containment" distro feature has been enabled
+  if the "process-containment" distro feature has been enabled
 * meta-bistro (https://github.com/Pelagicore/meta-bistro)
 
+The relationship with meta-bistro, which is also developed at Pelagicore, is
```
Comment (GordanM) [inline README.md]: Should the layer name be in back ticks (`meta-bistro`)? Have to check how its done in the other parts of the SWF.

File: README.md [hunk 2]
```diff
@@ -18,9 +18,14 @@ This layer depends on:
...
+that meta-bistro contains general software recipes, but no distribution and no
```
Comment (GordanM) [inline README.md]: I would add that `meta-bistro` adds IVi and development related recipes and changes to the PELUX image. For instance it adds the development tools packageconfig and the ivi packageconfig (have to verify this).

Comment (tobsan) [reply README.md]: meta-bistro does not change the PELUX image. Rather, the PELUX image makes use of things defined in meta-bistro.

I just checked, and it does create/append to some packagegroups, but the relation is still that PELUX  uses what's defined in bistro.

Comment (tobsan) [reply README.md]: That said, I think the comment about describing what it is that bistro brings rather than generic software, is a good one :)

**Comment by tobsan** [comment]
@GordanM updated after your comments.

**Pull request merged**